tests/acceptance/features/apiSpaces/spaceDownload.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Download space
+  As a user
+  I want to be able to download space
+  So that I can store it in my local storage
+
+
+  Scenario: user downloads a space
```

Comment:
@saw-jan what about adding test coverage for users tries to download space share with permission can edit and can manage? Will it be too much?

Reply:
should be easy with examples table

Reply:
But let's do it int next PR. This pr is almost a month old.

Scenarios to add:
- space member downloads space
- space-admin downloads other's space